fiboa_cli/convert_utils.py

```diff
@@ -545,3 +558,107 @@ def hash_df(df):
     buf = StringIO()
     df.info(buf=buf)
     return hash(buf.getvalue())
+
+
+class EsriRESTPregenerateConverterMixin:
```

Comment:
Can we put the two classes into separate files?

Reply:
Yes